plugins/modules/zos_job_submit.py

```diff
@@ -646,12 +639,12 @@ def run_module():
     if encoding is None:
         encoding = {
             "from": DEFAULT_ASCII_CHARSET,
-            "to": get_encoding_from_locale(module) or DEFAULT_EBCDIC_CHARSET,
+            "to": EncodeUtils().remote_charset(),
```

Comment:
`EncodeUtils().remote_charset()` has been removed in the latest update. I would recommend using `Defaults.get_default_system_charset()`

The default charset values has also been added to `encode.Defaults` class.

Reply:
whoops good catch @asifmahmud , will refer to your latest updates